pkg/config/cmd_config.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"io/ioutil"
+
+	"github.com/ghodss/yaml"
```

Comment:
we already use the `gopkg.in/yaml.v2` yaml package, no need to introduce another dependancy

Reply:
I don't want to introduce new dependency, but actually this package "github.com/ghodss/yaml" has been used in code if you check, so I think this one is an option too. And I have changed it to "gopkg.in/yaml.v2".